### ISSUE TITLE
Can we use this in windows?

### ISSUE THREAD

**Issue by tyluckyma** [muhlba91/icloud#10]
I'm using an iPhone, and wanna sync the calendar to my desktop which uses windows instead. May i ask do I have to use Mac to use you script or what?

**Comment by saqibomer** [comment]
It is a PHP script. You can run it on Windows/Mac OS/Linux. @tyluckyma

**Comment by muhlba91** [comment]
Hi,

if you have Java installed, you can download the GUI version (https://github.com/muhlba91/icloud/raw/master/groovy_java/cmdline-2.1.0.zip) and execute the batch script inside the 'bin' folder. Or if you prefer the PHP version, you can run it through a XAMPP (https://www.apachefriends.org/index.html) instance, for example. ;)

Hope this helps,
Daniel

**Comment by tyluckyma** [comment]
Yeah, I've tried both. But met the following problem. For the xampp I met:
![default](https://cloud.githubusercontent.com/assets/3775572/21526519/d33e80ee-cd5f-11e6-89fa-65ea22d385a5.png)

For the command line verison, I met:
![default](https://cloud.githubusercontent.com/assets/3775572/21526658/c376c2c4-cd60-11e6-9b10-e141a66e1bb8.png)

Any idea?

**Comment by tyluckyma** [comment]
I've tried to uncomment the "curl" in php.ini, but found the following problem:
![default](https://cloud.githubusercontent.com/assets/3775572/21526943/cb2b7cd8-cd62-11e6-8f81-cf98eca95fe8.png)

**Comment by muhlba91** [comment]
Hi,

re the Java cmdline issue: hat Java version are you running? :)
Because those zipped versions are compiled using Java 8, meaning if you want to run them with Java 7 or earlier you could get exactly this error. In case you are using, for example, Java 7 and don't want to upgrade the easiest solution is simply:
 - check out the folder https://github.com/muhlba91/icloud/tree/master/groovy_java/project
 - run `gradlew build distZip`
 - this creates the zip files too but compiled with your Java version which you can run then. ;)

**Comment by tyluckyma** [comment]
Awesome!!

Thanks for the reply. Upgrade Java to the latest version, and get the URL!!!

Already stared!

**Comment by muhlba91** [comment]
Hi,

they should support writing - at least they did till some time ago but who knows what Apple changes in the CalDAV standard. ^^
However, I am quite certain that they should adhere to the standardised way of writing events - can you try it with a different application (e.g. Sunbird) to check whether your used application does not send all the required information to the Apple CalDAV server? :)
